sematic/ui/packages/common/src/pages/RunSearch/filters/SearchTextSection.tsx

```diff
@@ -5,11 +5,12 @@ import { ResettableHandle } from "src/component/common";
 
 interface SearchTextSectionProps {
     onSearchChanged?: (search: string) => void;
+    defaultSearchString?: string | undefined;
 }
 
 const SearchTextSection = forwardRef<ResettableHandle, SearchTextSectionProps>((props, ref) => {
-    const { onSearchChanged } = props;    
-    const [search, setSearch] = useState<string>("");
+    const { onSearchChanged, defaultSearchString } = props;
+    const [search, setSearch] = useState<string>(!!defaultSearchString ? defaultSearchString : "");
```

Comment:
This is equivalent to `defaultSearchString  || ""`

Reply:
changed!